app/services/question_saver.rb

```diff
@@ -16,6 +16,19 @@ def save(form)
   private
 
   def persist_answers(form)
+    persist_room(form)
```

Comment:
I'm wondering if these 'special' answers should be stored in a separate place?

`SelectedAnswerStore` is now storing: 1) the answers given to questions by a user *AND* 2) specific answers + sor code that are sent to the API when 'creating' a repair, which feel like different things to me.

Reply:
I agree, but I'm not sure what the improved solution would look like yet. 

I feel like what we've got now doesn't actually have any problems yet, so it's good enough for now, but I think we should keep an eye open for problems and refactoring to avoid them when we find them.